### ISSUE TITLE
Suggestion: even distribution mode

### ISSUE THREAD

**Issue by billw2012** [algernon-A/LineToolLite#6]
Add a toggle next to the Spacing (in the same style as the dice one for random rotation) that will cause the tool to automatically increase the spacing to close the gap from the last placed item to the actually clicked end point. This would be particularly useful when 
Pseudocode:
```
float totalLineLength = (endPoint - startPoint).Length;

int itemsToPlace = (int)(totalLineLength / spacingAsSetInUI);
float remainder = totalLineLength - itemsToPlace * spacingAsSetInUI;

float finalSpacing = spacing + remainder / itemsToPlace;
```

Also another minor suggestion: sub 1 meter resolution for variable spacing and offset, perhaps 0.25m increments instead of 1m? For placing a line of trees for instance anything over 1m starts to look messy, and bushes are too small to use this option at all.

**Comment by algernon-A** [comment]
You can already get 0.1 meter increments by holding down the control key when pressing the increment button (shift key for 10m increments).

The even spacing is already under development, but I'll keep this here as a tracker.

**Comment by algernon-A** [comment]
Implemented in 1.0.10 5db1edd.
